website/src/css/custom.css

```
@@ -53,6 +53,20 @@ a:hover {
   color: blue;
 }
 
+/* Styling for the result of TOCInline
+ * The goal of this is to make the generated list look at similar to a Markdown list as possible
+ */
+div[class^='tableOfContentsInline'] ul {
+  padding-left: var(--ifm-list-left-padding);
+  list-style-type: revert;
+}
+
+div[class^='tableOfContentsInline'] li {
+  margin: revert;
+}
+div[class^='tableOfContentsInline'] li + li {
```

Might just be a question of personal taste, but I find it more readable to write this selector as

```suggestion
div[class^='tableOfContentsInline'] li:not(:first-child) {
```

They're technically not equivalent, but I'm assuming you are using the sibling selector (`+`) in order to select everything but the first child.

---

I will sanity check that it gives the same result, but I agree that is better.

---

Looks the same, thanks!